tensorflow/lite/micro/tools/ci_build/test_bluepill.sh

```diff
@@ -34,12 +34,12 @@ readable_run make -f tensorflow/lite/micro/tools/make/Makefile TARGET=${TARGET}
 
 # check that the release build is ok.
 readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
-readable_run make -j8 -f tensorflow/lite/micro/tools/make/Makefile TARGET=${TARGET} OPTIMIZATION_LEVEL=-O3 BUILD_TYPE=release build
+readable_run make -j8 -f tensorflow/lite/micro/tools/make/Makefile TARGET=${TARGET} BUILD_TYPE=release build
 
 # Next, build w/o release so that we can run the tests and get additional
 # debugging info on failures.
 readable_run make -f tensorflow/lite/micro/tools/make/Makefile clean
-readable_run make -j8 -f tensorflow/lite/micro/tools/make/Makefile TARGET=${TARGET} OPTIMIZATION_LEVEL=-Os test
```

Comment:
This line is now doing the same as then next one. Let's remove one of them. I'm ok switching bluepill to only use the default optimization levels.

Reply:
Done.